pkg/app/piped/executor/kubernetes/kubernetes.go

```diff
@@ -88,6 +88,11 @@ func (e *deployExecutor) Execute(sig executor.StopSignal) model.StageStatus {
 		return model.StageStatus_STAGE_FAILURE
 	}
 
+	chartRepoName := e.deployCfg.Input.HelmChart.Repository
+	if chartRepoName != "" {
+		e.deployCfg.Input.HelmChart.Insecure = ds.DeploymentConfig.PipedSpec.IsInsecureChartRepository(chartRepoName)
```

Comment:
`DeploymentConfig` is only for deployment configuration not Piped configuration so `ds.DeploymentConfig.PipedSpec` is always nil.
Instead of that, you can have Piped config with `e.PipedConfig` because it is placing inside `executor.Input`.

Reply:
I'm sorry for the delay. I've been a little busy, so I'm sorry for the late reply.
I was wondering how to pass on the values, thank you for telling me about.

Reply:
👍